_src/pattern-library/sass/components/_headings.scss

```diff
@@ -138,3 +138,9 @@ $headings-count: 8;
         @extend %hd-#{$i};
     }
 }
+
+// broad stroke extend for emphasized heading
+%headings-emphasized {
```

Comment:
Similar to the copy-based feedback, can you move this up to the ``#GLOBAL`` section of the headings partial and then extend it in place of the [canned rules here](https://github.com/edx/ux-pattern-library/blob/frances/emphasized-extends/_src/pattern-library/sass/components/_headings.scss#L28)?